### PR TITLE
MYST-220 Return proposals only for active nodes

### PR DESCRIPTION
**Pull request by donce**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -24,9 +25,12 @@ def __init__(self, node_key):
...
         self.created_at = datetime.utcnow()
 
-    def get_status(self):
-        # TODO: implement status checking
-        return 'active'
+    def is_active(self):
+        last_ping = self.updated_at or self.created_at
+        return last_ping > datetime.utcnow() - NODE_AVAILABILITY_TIMEOUT
+
+    def update_timestamp(self):
```
Comment (interro) [inline models.py]: this method can be reused in register_proposal() too

Comment (donce) [reply models.py]: Done.

File: app.py [hunk 2]
```diff
@@ -126,7 +126,7 @@ def register_proposal(caller_identity):
     node.ip = request.remote_addr
     node.country = detect_country(node.ip) or ''
     node.proposal = json.dumps(proposal)
-    node.updated_at = datetime.utcnow()
+    node.update_timestamp()
```
Comment (Waldz) [inline app.py]: Timestamp is too abstract, could be several dates.
`markNodeActive()`?

File: app.py [hunk 3]
```diff
@@ -143,8 +143,9 @@ def proposals():
     else:
         nodes = Node.query.all()
 
+    active_nodes = [node for node in nodes if node.is_active()]
```
Comment (Waldz) [inline app.py]: Nodes should be filtered in query layer
+
Dashboard *active filter* logic now is duplicated. Could be moved to reusable helpers

File: app.py (hunk 2 above)
Comment (donce) [reply app.py]: Updated.

File: app.py (hunk 3 above)
Comment (donce) [reply app.py]: Re-used database-level filtering.

**Pull request merged**